File: 3.javaTutorials/RAHUL5.java
import java.util.Scanner;

public class RAHUL5 {
    public static void main(String[] args) {
        try (Scanner Sc = new Scanner(System.in)) {
            System.out.println("give first number here");
            int a = Sc.nextInt();
            System.out.println("give second number here");
            int b = Sc.nextInt();
            System.out.println(a + b);
            System.out.println("give first number here");
            float c = Sc.nextFloat();
            System.out.println("give second number here");
            float d = Sc.nextFloat();
            System.out.println(c + d);
            boolean e = Sc.hasNextInt();
            System.out.println(e);

            // String str = Sc.next();
            // System.out.println(str);

            // String str2 = Sc.nextLine();
            // System.out.println(str2);
        }
    }
}

File: 8.HenceforthInternship/typescript/src/rahul5.ts
const sumfun = (a: number, b: number = 10, c?: number): number => {
    console.log(c)
    return c?a + b+c:a+b;
}
console.log(sumfun(3));
console.log(sumfun(10, 2));
console.log(sumfun(10, 2,5));
export{}
